custom_components/google_home/sensor.py

```diff
@@ -179,7 +196,7 @@ def device_state_attributes(self):
     def _get_timers_data(self):
         """Update timers data extracting it from coordinator"""
         device = self.get_device()
-        return device.get_sorted_timers_as_dict()
+        return self.as_dict(device.get_sorted_timers())
```

Comment:
It seems this can be removed. `"timers": device.get_sorted_timers()` should be enough.

Reply:
Yeah, I see. Let's keep it for now then.